diagrams/lib/main.dart

```diff
@@ -67,4 +67,5 @@ Future<Null> main(List<String> arguments) async {
     }
   }
   print('Total elapsed time: ${new DateTime.now().difference(start)}');
+  exit(0);
```

Comment:
?

Reply:
In order for the generator script to continue and download the generated diagrams, the app needs to actually exit, or the "flutter run" that it invokes hangs.

Reply:
Interesting. I wonder if this means we can't run this on iOS?

Reply:
Well, we can't run it on iOS anyhow because I used adb to transfer the resulting images back to the host anyhow. Is running on iOS important?

Reply:
ah yeah, good point. We should just mention that in the README somewhere.